src/core/services/theming/colors.js

```diff
@@ -0,0 +1,129 @@
+(function () {
+    "use strict";
+
+    //module global to keep an reference to the dynamic style sheet
+    var customSheet;
```

Comment:
This doesn't need to be a "module global" (with a minor 2-lines refactoring).
Module globals aren't that much better than globals anyway :smile: